tests/ut/py/test_task_interface.py

```
@@ -465,6 +466,192 @@ def test_no_capacity_limit_scalars(self):
+        desc = sidecar.tensors[0].desc
+        assert desc.address_space == RemoteAddressSpace.REMOTE_DEVICE
+        assert desc.owner_endpoint_id == 3
+        assert desc.buffer_id == 11
+        assert desc.generation == 2
+        assert desc.offset == 8
+        assert desc.nbytes == 4
+        assert desc.remote_addr == 0xCAFE
+        assert desc.rkey_or_token == 0xBEEF
+
+    def test_remote_buffer_handle_is_opaque_to_public_constructor(self):
+        with pytest.raises(TypeError, match="Worker.remote_malloc"):
```

Ruff (0.15.17)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)